assignments/objects.js

```diff
@@ -11,54 +11,109 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: "Example",
+  email: "examples@you.edu",
+  gender: "F"
+};
 
 // Write your intern objects here:
-
-
-// ==== Challenge 2: Reading Object Data ==== 
```

Comment:
these objects are not working for me. When I plug this into repl.it it tells me that example doesn't work. Now I understand from reading the code what you're trying to do. But your answer here is a little more "sudo code" than real code. Look at the example how it uses curly brackets to enclose the info in the object. 

I like where your heads at, you're trying to do something more advance than this assignment asks for, but it's causing your answers to not work on my end. 

Keep this in mind when you do something like your Sprint at the end of the week.